src/ui/LiveSplit.tsx

```diff
@@ -39,6 +40,13 @@ type Menu =
     { kind: MenuKind.SettingsEditor, config: HotkeyConfig } |
     { kind: MenuKind.About };
 
+export interface Props {
+    splits: string,
+    layout: any,
+    settings: any,
```

Comment:
These probably shouldn't be any.

Reply:
In fact I believe that we even should "ban any". They somewhat recently introduced `unknown` as a [top type](https://en.wikipedia.org/wiki/Top_type) that doesn't imply dynamic typing.